src/components/CreateCoursePage/index.jsx

```diff
@@ -112,11 +122,11 @@ CreateCoursePage.propTypes = {
   }),
   publisherUserInfo: PropTypes.shape({
     organizations: PropTypes.array,
-    error: PropTypes.string,
+    error: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Shouldn't `publisherUserInfo` have similar code in terms of putting it in an array for that error alert?